### DISCUSSION TITLE
Monorepo with multiple Kustomizations.

### DISCUSSION THREAD

**Discussion by netthier** [fluxcd/flux2#2777] [Q&A] [answered]
I'm very new to Flux and am currently attempting to deploy cert-manager and a set of ClusterIssuers using the monorepo directory structure described [here](https://fluxcd.io/docs/guides/repository-structure/#repository-structure).
I have a Kustomization pointing at the `infrastructure` directory, which contains the cert-manager and other apps like sealed-secrets and ingress-nginx.
I encountered this problem while doing so : https://github.com/fluxcd/flux2/issues/2243
While I understand the solution presented, I'm unsure where to place the new Kustomizations, especially since I already have one pointing at the `infrastructure/` directory.
Is there a demo of such a repo with a cert-manager deployment?

**Comment by stefanprodan** [comment, marked as answer]
Create a dir next to `infrastructure` e.g. `infra-config`, place there the ClusterIssuers, then create a Flux Kustomization for that dir that has `dependsOn` for infrastructure.

Here is an example but with Istio https://github.com/stefanprodan/gitops-istio/blob/main/clusters/my-cluster/istio.yaml#L26

**Reply by netthier** [reply]
Got it to work that way, thanks :+1:
